flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -392,6 +394,25 @@
                     .withDescription(
                             "Configure the minimum increase in parallelism for a job to scale up.");
 
+    @Documentation.Section({
+        Documentation.Sections.EXPERT_SCHEDULING,
+        Documentation.Sections.ALL_JOB_MANAGER
+    })
+    public static final ConfigOption<Duration> RESOURCE_WAIT_TIMEOUT =
+            key("jobmanager.declarative-scheduler.resource-wait-timeout")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(10))
```

Comment:
aren't the k8s e2e tests failing because 10 seconds is too aggressive. Maybe set the default to 30 ?

Reply:
For now I intend to set these explicitly in the e2e tests, and would wait with increasing the defaults once we gathered more data.

Overall, my impression for the e2e tests is that we should provide a way to disabling the downScaling, in cases where you absolutely need to run a job with a certain parallelism. (which the e2e do)